.gitignore

```diff
@@ -271,6 +271,5 @@ lib/msal-core/lib-es6
 *.zip
 samples/**/screenshots
 samples/**/data/cache.json
-samples/**/data/testCache.json
```

Comment:
The first test was always timing out because the test was being executed before the code that created the cache file. I tried pretty much everything in terms of reformatting the cache accessors and test utilities to use callbacks instead of async/await but nothing worked. Having the file already exist by the time the tests were executed solved it. We use the setup and teardown code to reset it before and after tests so these files should not be a problem.

Reply:
Put it back since the cache file creation seems to be fixed now.